src/styles/html.scss

```diff
@@ -0,0 +1,8 @@
+body {
```

Comment:
`html.scss` is not really good file name, since you reset default styles you can rename it for example to `normalize.scss`, or move this styles to `main.scss`